prov/rxm/src/rxm.h

```diff
@@ -441,7 +441,7 @@ struct rxm_ep {
 	struct fi_info 		*msg_info;
 	struct fid_pep 		*msg_pep;
 	struct fid_eq 		*msg_eq;
-	struct slist		msg_eq_entry_list;
+	struct slistfd		msg_eq_entry_list;
```

Comment:
Can you work on the wording for this commit message?  The link between an EQ and the CQ isn't obvious.

Reply:
Ok. I'll add a note on why CQ progress is dependent on EQ progress.